tests/FakeItEasy.Specs/ResetFakeSpecs.cs

```diff
@@ -3,6 +3,7 @@
 using System;
 using FakeItEasy.Core;
 using FluentAssertions;
+using Tests.TestHelpers;

```

Comment:
Man, the build is mad about this one.

Reply:
Oops! We might be able to configure this in .editorconfig so that the IDE and StyleCop agree with each other...